README.md

```diff
@@ -4,22 +4,94 @@ This project is a ActiveMQ Artemis Self Provisioning Plugin to the Administrator
 
 ## Local development
 
+To be able to run the local development environment you need to:
+* have access to an OpenShift cluster
+* have the operator installed on the cluster
+* have the plugin running
+* have the console running
+
+### Setting up an OpenShift cluster
+
+In order to run the project you need to have access to an OpenShift cluster.
+If you don't have an access to a remote one you can deploy one on your machine
+with `crc`.
+
+#### Local cluster
+
+Follow the documentation:
+https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.34/html-single/getting_started_guide/index#introducing
+
+> [!WARNING]
+> If you're running the cluster on Fedora CSB, note that [you might
```

Comment:
For more folks to understand the context here, perhaps we should be saying something like 'if you see symptom X and/or symptom Y on startup' then see issue Z, this specific comment https://github.com/crc-org/crc/issues/4110#issuecomment-2044707940 for configuration to try.

It's likely that reference to 'Fedora CSB' will prompt searches of google to grok the context.